src/bin/edit/localization.rs

```diff
@@ -695,6 +736,7 @@ const S_LANG_LUT: [[&str; LangId::Count as usize]; LocId::Count as usize] = [
         /* it      */ "Il testo copiato è troppo grande per essere condiviso.",
         /* ja      */ "コピーしたテキストは大きすぎて共有できません。",
         /* ko      */ "복사한 텍스트가 너무 커서 공유할 수 없습니다.",
+        /* pl      */ "Skopiowany tekst jest zbyt duży, aby można go było przesłać.",
```

Comment:
```suggestion
        /* pl      */ "Skopiowany tekst jest zbyt duży, żeby go przesłać.",
```
(optional, spells/reads quicker)